hal/lib/athena/HALAthena.cpp

```diff
@@ -5,8 +5,6 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "HAL/HAL.h"
```

Comment:
For every header file, we need to have at least one cpp file that includes it first to ensure the header is standalone.